CHANGELOG.md

```diff
@@ -116,7 +116,12 @@ may elicit a deprecation warning. See [#9086] for more details.
 
 </details>
 
-## v2.5.0 - TBD
+## v2.6.0 - TBD
+
+## v2.5.0 - 2022-12
```

Comment:
Remember that the "update CHANGELOG" PR should now also include an edit to CMakeLists.txt to set PROJECT_VERSION_PRE_RELEASE to the empty string.